torch/optim/optimizer.py

```diff
@@ -76,7 +77,7 @@ def _use_grad(self, *args, **kwargs):
             # see https://github.com/pytorch/pytorch/issues/104053
             torch.set_grad_enabled(self.defaults["differentiable"])
             torch._dynamo.graph_break()
-            ret = func(self, *args, **kwargs)
+            ret = func(*args, **kwargs)
```

Comment:
wait why this change

Reply:
I tried using typing.Concatenate to type self : Optimizer, but combined with Callable[Concatenate] this require a lot fo extra type varing or int he easy case of Concatenate surprisingly did not allow optimizer subclasses (defeating the whole point)